scripts/feedback_aggregator.py

```diff
@@ -143,6 +145,14 @@ def args_parser(args: list[str]) -> argparse.Namespace:
         action="store_true",
         help="Perform check if Ceph bucket is accessible",
     )
+    parser.add_argument(
+        "-i",
+        "--ignore",
+        # cluster UUID used by CCX monitoring infrastructure
+        default="00000000-1111-0000-1111-000000000001",
+        help="Cluster IDs to ignore during processing splited by comma: "
+        + "00000000-0000-0000-0000-000000000001, 00000000-0000-0000-0000-000000000002",
```

Comment:
@tisnik you show a space after the comma here but I assume that is incorrect (if a space is present it will be treated as the next arg)